### ISSUE TITLE
Error running exsomiser: Error starting ApplicationContext

### ISSUE THREAD

**Issue by rhassnan** [exomiser/Exomiser#428]
I am trying to run exomiser analysis using the test-analysis-exome.yml. It works fine when I use the example provided by Exomiser but when I use my own VCF files it produces this error: 
-------------------------------------------

2022-03-17 10:30:46.555  INFO 116268 --- [           main] o.m.e.core.genome.VariantFactoryImpl     : Annotating variant records, trimming sequences and normalising positions...
2022-03-17 10:30:50.975  INFO 116268 --- [           main] o.m.e.c.analysis.AbstractAnalysisRunner  : Loaded 100000 variants - 1627 passed variant filters...
2022-03-17 10:30:54.106  INFO 116268 --- [           main] o.m.e.core.genome.VariantFactoryImpl     : Processed 191948 variant records into 192294 single allele variants (including 0 structural variants)
2022-03-17 10:30:54.106  INFO 116268 --- [           main] o.m.e.core.genome.VariantFactoryImpl     : Variant annotation finished in 0m 7s 550ms (7550 ms)
2022-03-17 10:30:54.107  INFO 116268 --- [           main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-03-17 10:30:54.114 ERROR 116268 --- [           main] o.s.boot.SpringApplication               : Application run failed

java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:794)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:345)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1343)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1332)
	at org.monarchinitiative.exomiser.cli.Main.main(Main.java:52)
Caused by: org.monarchinitiative.svart.CoordinatesOutOfBoundsException: Fully-closed coordinates 3:198023180-198023180 out of contig bounds [1,198022430]
	at org.monarchinitiative.svart.Coordinates.validateCoordinates(Coordinates.java:197)
	at org.monarchinitiative.svart.BaseGenomicRegion.<init>(BaseGenomicRegion.java:23)
	at org.monarchinitiative.svart.BaseVariant.<init>(BaseVariant.java:20)
	at org.monarchinitiative.svart.impl.DefaultVariant.<init>(DefaultVariant.java:8)
	at org.monarchinitiative.svart.impl.DefaultVariant.of(DefaultVariant.java:23)
	at org.monarchinitiative.svart.Variant.of(Variant.java:85)
	at org.monarchinitiative.svart.util.VcfConverter.convert(VcfConverter.java:60)
	at org.monarchinitiative.exomiser.core.genome.VariantContextConverter.convertToVariant(VariantContextConverter.java:106)
	at org.monarchinitiative.exomiser.core.genome.VariantFactoryImpl.buildVariantEvaluations(VariantFactoryImpl.java:161)
	at org.monarchinitiative.exomiser.core.genome.VariantFactoryImpl.lambda$buildAlleleVariantEvaluations$1(VariantFactoryImpl.java:110)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList$SubList$2.forEachRemaining(ArrayList.java:1481)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
	at java.base/java.util.stream.ReferencePipeline$15$1.accept(ReferencePipeline.java:541)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at org.monarchinitiative.exomiser.core.analysis.AbstractAnalysisRunner.loadAndFilterVariants(AbstractAnalysisRunner.java:197)
	at org.monarchinitiative.exomiser.core.analysis.AbstractAnalysisRunner.run(AbstractAnalysisRunner.java:113)
	at org.monarchinitiative.exomiser.core.Exomiser.run(Exomiser.java:83)
	at org.monarchinitiative.exomiser.core.Exomiser.run(Exomiser.java:69)
	at org.monarchinitiative.exomiser.cli.ExomiserCommandLineRunner.runJob(ExomiserCommandLineRunner.java:79)
	at org.monarchinitiative.exomiser.cli.ExomiserCommandLineRunner.runJobs(ExomiserCommandLineRunner.java:62)
	at org.monarchinitiative.exomiser.cli.ExomiserCommandLineRunner.run(ExomiserCommandLineRunner.java:57)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:791)
	... 5 common frames omitted

--------------------------
Can someone help me with this error ? 
Thank you

**Comment by ielis** [comment]
Hi @rhassnan,
the line 
```
Caused by: org.monarchinitiative.svart.CoordinatesOutOfBoundsException: Fully-closed coordinates 3:198023180-198023180 out of contig bounds [1,198022430]
```
indicates that your input VCF has a variant located at position `3:198,023,180`. 

However, contig `3` of the genomic assembly that you're working with (I'm assuming it's _GRCh37_) is only `198,022,430` bases long (check [here](https://ftp.ncbi.nlm.nih.gov/genomes/all/annotation_releases/9606/105.20220307/GCF_000001405.25_GRCh37.p13/GCF_000001405.25_GRCh37.p13_assembly_report.txt)). 

So it seems like there is error in your input VCF where a variant is located past end of the contig.

**Comment by rhassnan** [comment]
Hi @ielis 
Thank you I changed the genomic assembly to hg38 and it working now. 
Thank you for helping !
